Trippy/View Controllers/Saved Items Tab/SegmentedViewController.m

```diff
@@ -0,0 +1,107 @@
+//
+//  SegmentedViewController.m
+//  Trippy
+//
+//  Created by Catherine Lu on 8/2/22.
+//
+
+#import "SegmentedViewController.h"
+#import "LoginViewController.h"
+#import "LogoutHandler.h"
+#import "SceneDelegate.h"
+#import "SavedItinerariesViewController.h"
+#import "SavedCollectionsViewController.h"
+
+@interface SegmentedViewController () <LogoutHandlerDelegate>
+@property (strong, nonatomic) LogoutHandler *logoutHandler;
+@property (weak, nonatomic) IBOutlet UIView *contentView;
+@property (weak, nonatomic) IBOutlet UISegmentedControl *segmentedControl;
+
+@property (strong, nonatomic) SavedItinerariesViewController *itVC;
+@property (strong, nonatomic) SavedCollectionsViewController *colVC;
+@property (strong, nonatomic) UIViewController *currentVC;
+@end
+
+@implementation SegmentedViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Set up Parse interface
+    self.logoutHandler = [[LogoutHandler alloc] init];
+    self.logoutHandler.delegate = self;
+    
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
+    self.itVC = [storyboard instantiateViewControllerWithIdentifier:@"SavedItinerariesViewController"];
+    self.colVC = [storyboard instantiateViewControllerWithIdentifier:@"SavedCollectionsViewController"];
+    
+    [self displayCurrentTab:self.segmentedControl.selectedSegmentIndex];
+}
+
+- (void)viewWillDisappear:(BOOL)animated {
+    [super viewWillDisappear:animated];
+    if (self.currentVC) {
+        [self.currentVC viewWillDisappear:NO];
+    }
+}
+
+- (UIViewController *)viewControllerForSelectedSegmentIndex:(int)index {
+    switch (index) {
+        case 0:
```

Comment:
nit: I think you could make this a bit cleaner by using an enum with the numeric values hardset (something like ITINERARY_VIEW = 0, COLLECTION_VIEW = 1, etc.)